### PR TITLE
Add font-awesome; Add support links

### PR DESCRIPTION
**Pull request by digitalentity**

### CODE REVIEW

**Comment by skaman82** [comment]
The Slack/RCGruoups icons are also bit too big :) font-size: 35px should be enough

File: src/css/tabs/landing.css [hunk 1]
```diff
@@ -18,6 +18,36 @@
...
+.tab-landing .community ul {
+    font-size: 20px;
+    text-align: center;
+    width: 400px;
+    height: 50px;
+    display: inline-block;
+    position: relative;
+}
+
+.tab-landing .community ul i {
```
Comment (skaman82) [inline src/css/tabs/landing.css]: font-size: 35px;
color: white;
margin-left: -200px;
margin-top: 5px;

**Comment by digitalentity** [comment]
@DzikuVx what do you think about this PR?

**Comment by DzikuVx** [comment]
Go ahead

**Pull request merged**